.github/workflows/docker-test-containers.yml

```diff
@@ -11,13 +11,13 @@ jobs:
       matrix:
         include:
           - source: jaegertracing/all-in-one:1.17
-            target_tag: jaeger
-          - source: otel/opentelemetry-collector-contrib-dev:latest
-            target_tag: otel-collector-dev
+            target_image: jaeger
+          - source: otel/opentelemetry-collector-contrib:latest
```

Comment:
@jkwatson I also switched from -dev to normal as we discussed before not targeting it for our build I think